tests/testsuite/bench.rs

```diff
@@ -314,9 +314,9 @@ fn cargo_bench_failing_test() {
 [RUNNING] [..] (target/release/deps/foo-[..][EXE])",
         )
         .with_stdout_contains("[..]thread '[..]' panicked at[..]")
-        .with_stdout_contains("[..]assertion failed[..]")
-        .with_stdout_contains("[..]left: `\"hello\"`[..]")
-        .with_stdout_contains("[..]right: `\"nope\"`[..]")
+        .with_stdout_contains("[..]assertion [..]failed[..]")
```

Comment:
thanks @weihanglo!  Does `[..]` match `.*` or `.+` (in regex terms)?

Reply:
Yes, similar to `.*` for a line. Please see <https://doc.rust-lang.org/nightly/nightly-rustc/cargo_test_support/compare/index.html>.

I am doing this because it need to match both stable and nightly formats 🥲